package.json

```diff
@@ -27,7 +27,7 @@
     "react": "^19.1.0",
     "react-18": "npm:react@^18.3.1",
     "react-dom": "^19.1.0",
-    "react-dom-18": "patch:react-dom@npm%3A18.3.1#~/.yarn/patches/react-dom-npm-18.3.1-a805663f38.patch",
+    "react-dom-18": "npm:react-dom@^18.3.1",
```

Comment:
This still works nicely (http://localhost:4321/experiments/react-extension/hydration-18-root-nothing):

Without the patch | With the patch
-|-
<img width="1512" height="828" alt="image" src="https://github.com/user-attachments/assets/8b9246e1-c954-408f-b7ea-ea955bb2617f" /> | <img width="1512" height="827" alt="image" src="https://github.com/user-attachments/assets/72b713dd-d6ae-408c-8974-fb61f5acdb79" />